workalendar/tests/test_europe.py

```diff
@@ -835,6 +836,45 @@ def test_year_2018(self):
         self.assertIn(date(2018, 11, 4), holidays)  # Day of Unity
 
 
+class UkraineTest(GenericCalendarTest):
+    cal_class = Ukraine
+
+    def test_year(self):
+        holidays = self.cal.holidays_set(2010)
+        self.assertIn(date(2010, 1, 1), holidays)  # New Year's Day
+        self.assertIn(date(2010, 1, 7), holidays)  # Christmas (Orthodox)
+        self.assertIn(date(2010, 3, 8), holidays)  # International Women's Day
+        self.assertIn(date(2010, 5, 1), holidays)  # Labour Day
+        self.assertIn(date(2010, 5, 9), holidays)  # Victory Day
+        self.assertIn(date(2010, 5, 24), holidays)  # Whit Monday
+        self.assertIn(date(2010, 6, 28), holidays)  # Constitution Day
+        self.assertIn(date(2010, 8, 24), holidays)  # Day of Unity
+        self.assertIn(date(2010, 12, 25), holidays)  # Christmas
```

Comment:
Catholic Christmas is a holiday only starting from the 2017

Reply:
Fixed it and updated the tests